rails-commenting.rb

```diff
@@ -71,6 +71,6 @@ def blog_post_params
 # FILE: app/models/blog_post.rb
 
 class BlogPost < ApplicationRecord
-  # 10)
+  # 10)This is linked to another model called comments 
```

Comment:
Correct its linked to a comments model and the comments model would have belongs_to :BlogPost_id